src/map/atcommand.c

```diff
@@ -1714,6 +1714,15 @@ ACMD(bodystyle)
 
 	memset(atcmd_output, '\0', sizeof(atcmd_output));
 
+	if (!((sd->job&MAPID_THIRDMASK) == MAPID_GUILLOTINE_CROSS || (sd->job&MAPID_THIRDMASK) == MAPID_GENETIC
```

Comment:
Uhm, you may verify the return types here - http://icecube.wisc.edu/~dglo/c_class/operators.html

If you're assuming that boolean is a type, you might as well accept it for its real values as false (0) and true (1). All of the boolean expressions return an integer 0 or 1. CPU registers for conditional flags are set as 0 or 1 and not true or false.

The expression - `(sd->job&MAPID_THIRDMASK) == MAPID_GUILLOTINE_CROSS` for example would return 
1) `((int) == int)`
2) `(1 or 0)` and not `(true or false)`

And so the statement above isn't wrong, neither is `() == false` or `() == true` so long as they are both equal to either 0 and 1 respectively.